docs/eventing/samples/ping-source/README.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "PingSource example"
+linkTitle: "PingSource"
+weight: 10
+type: "docs"
+---
+
+PingSource example shows how to configure PingSource as event source for
+functions.
+
+## Before you begin
+
+1. Setup [Knative Serving](../../../serving).
+1. Setup [Knative Eventing](../../../eventing).
+
+## Create a Knative Service
+
+In order to verify `PingSource` is working, we will create a simple Knative
+Service that dumps incoming messages to its log.
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+spec:
+  template:
+    spec:
+      containers:
+        - image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/event_display
+```
+
+Use following command to create the service from `service.yaml`:
+
+```shell
+kubectl apply --filename service.yaml
+```
+
+## Create a PingSource
+
+For each set of ping events you want to request, you need to create an Event
+Source in the same namespace as the destination. If you need a different
+ServiceAccount to create the Deployment, modify the entry accordingly in the
+yaml.
+
+```yaml
+apiVersion: sources.knative.dev/v1alpha1
+kind: PingSource
+metadata:
+  name: test-ping-source
+spec:
+  schedule: "*/2 * * * *"
+  data: '{"message": "Hello world!"}'
+  sink:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: event-display
+```
+
+Use following command to create the event source from `ping-source.yaml`:
+
+```shell
+kubectl apply --filename ping-source.yaml
+```
+
+## Verify
+
+We will verify that the message was sent to the Knative eventing system by
```

Comment:
```suggestion
Verify that the message was sent to the Knative eventing system by
```

When possible, use the second person "you" or imperative. This makes it clearer when the user needs to take action. Also note that "you" occurs a few lines down, so this will be more consistent as well.